src/pages/posts/index.js

```diff
@@ -32,8 +32,9 @@ export default function Page() {
       <Header title="Latest Posts" />
 
       <Main className="container">
-        <Posts posts={data?.nodes} readMoreText={'Read More'} id="posts-list" />
+        <Posts posts={data?.nodes} readMoreText="Read More" id="posts-list" />
         <LoadMore
+          className="text-center"
```

Comment:
Would it make sense to elevate this to the `LoadMore` component level?

Reply:
You mean have a div above the LoadMore button with a `className="text-center"` ?

Reply:
I'm wondering if only this particular instance of `LoadMore` needs centered text or if all `LoadMore` components should have centered text. If all `LoadMore` need centered text, then we could give that responsibility to the component instead of passing a className every time. Would a `text-align: center` [here](https://github.com/wpengine/atlas-blueprint-portfolio/blob/200d46db28fecdfb0401653a66f6f01abcf88126/src/components/LoadMore/LoadMore.module.scss#L1) work?

Reply:
I wouldn't want to pass text-alignment to the component itself just in case we want to drop it in a place where it shouldn't have to be centered.